packages/ra-core/src/core/CoreAdminRoutes.tsx

```diff
@@ -135,6 +136,10 @@ export const CoreAdminRoutes = (props: CoreAdminRoutesProps) => {
                                         ) : null
                                     }
                                 />
+                                <Route
+                                    path="/authentication-error"
```

Comment:
I don't understand why we need this page - no hook ever redirects to it.

Reply:
As discussed, it will be used in the next PR. Added now by mistake